pkg/mfa/client.go

```diff
@@ -294,3 +304,80 @@ func (c *Client) VerifyChallenge(
 	}
 
 }
+
+// DeleteFactor deletes a Factor.
```

Comment:
nit: change comment to `Deletes an authentication factor` to be in line with other methods.